<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Тестовый пример</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>641</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="filterWrap">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="1" column="1">
      <widget class="QComboBox" name="select_employee">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Выберите сотрудника</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="select_departments">
       <property name="whatsThis">
        <string/>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Выберите отдел</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выберите отдел *</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Выберите сотрудника</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checker_poib">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>ПОИБ СОБИ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checker_eis">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>ЕИС</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checker_axiok">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Аксиок Планирование</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeView" name="outView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="submit">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Получить данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Очистить вывод</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
